src/lexeme.js

```diff
@@ -62,6 +62,25 @@ class Lexeme {
       this.inflections.length > 0
   }
 
+  /**
+   * disambiguate a Lexeme with a supplied Lexeme
+   * @param {Lexeme} lexeme the lexeme to use for disambiguation
+   */
+  disambiguate (lexeme) {
+    if (this.lemma.isEqual(lexeme.lemma)) {
+      let keepInflections = []
+      // iterate through this lexemes inflections and keep only thoes that are disambiguatedBy by the supplied lexeme's inflection
+      for (let inflection of this.inflections) {
+        if (inflection.disambiguatedBy(lexeme.inflections[0])) {

```

Comment:
Because in my use case, the homonym that is used as a disambiguator specifies that it must be X inflection, so it will have only one.  But, I see your point, particularly given your comments on the Components PR about the disambiguate method in Homonym, and think the code will be more flexible if we assume we might have a narrowing down from multiple possibilities, to a few, to a single one.  I will go ahead and change this code accordingly. Thank you!